docs/pages/advanced/index.js

```diff
@@ -33,6 +34,20 @@ export default function Advanced() {
       {md`
       # Advanced
 
+      ## Accessibility
```

Comment:
TODO: More feedback from maintainers about how accessibility and the new api should be addressed.